app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+## class Post
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  has_many :comment
+  has_many :like
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_posts_counter
+    author.increment!(:posts_count)
+  end
```

Comment:
- [ ] [OPTIONAL] Good job on this method, however we can go further by including a [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save`[Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call this method anytime a post is saved to the database.